packages/dd-trace/test/setup/mocha.js

```diff
@@ -96,7 +97,7 @@ function withNamingSchema (
               expect(span).to.have.property('service', expectedServiceName)
             })
             .then(done)
-            .catch(done)
+            .catch(err => err && done(err))
```

Comment:
Are the promises failing with falsey errors? This looks like a catch fires but that it might not trigger the end of the test. These two lines might be better as:

```js
            .catch(err => done(err || 'rejected without reason'))
```